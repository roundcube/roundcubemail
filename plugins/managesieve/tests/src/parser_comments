require ["envelope","fileinto","imap4flags","subaddress"];

# simple to-be-ignored comment with an empty line after (#5657)

# another comment
if envelope :detail "to" "addressextension"
{
	setflag "\\Flagged";
}

# Spam
if header :is "X-Spam-Flag" "YES"
{
	fileinto "Junk";
	stop;
}
