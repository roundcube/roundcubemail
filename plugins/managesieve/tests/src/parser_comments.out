require ["envelope","fileinto","imap4flags","subaddress"];
if envelope :detail "to" "addressextension"
{
	setflag "\\Flagged";
}
if header :is "X-Spam-Flag" "YES"
{
	fileinto "Junk";
	stop;
}
